app/src/main/java/com/ichi2/apisample/MainActivity.java

```diff
@@ -197,6 +211,12 @@ public void onClick(View v) {
                     mAnkiDroid.requestPermission(MainActivity.this, AD_PERM_REQUEST);
                     return;
                 }
+                if (!MainActivity.this.doesModelExist() ||
```

Comment:
Why do you use MainActivity.this.*() method calls sometimes? Sometimes just this.*() and sometimes just *()... Is there any difference?
If no, it makes sense to use single style.